StreamChatUITestsAppUITests/Tests/Performance/ScrollTime.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import XCTest
+
+@available(iOS 15.0, *)
+class ScrollTime: StreamTestCase {
+    
+    func testMessageList() {
+        WHEN("user opens the message list") {
+            backendRobot.generateChannels(count: 1, messagesCount: 200)
+            userRobot.login().openChannel()
+        }
+        THEN("user scrolls the message list") {
+            measure(metrics: [XCTOSSignpostMetric.scrollingAndDecelerationMetric]) {
+                userRobot.scrollMessageListUp()
+            }
+        }
+    }
+    
+    func testChannelList() {
+        WHEN("user opens the channel list") {
+            backendRobot.generateChannels(count: 200)
```

Comment:
Same here, 200 is not realistic to have, since we also have limit on the backend of 25 (I think). We could try the same pagination thing, if possible.

Reply:
Hmm, I don't get it. In this test we generate a channel with a history of 200 messages. When user opens the channel and scrolls it, paginations goes into play and loads new pages after every 25 messages.

Reply:
discussed over Slack